innopoints/views/random.py

```diff
@@ -0,0 +1,35 @@
+"""Return a random page.
+
+- GET /random_page
+"""
+
+import random
+
+from flask import jsonify
+
+from innopoints.blueprints import api
+from innopoints.extensions import db
+from innopoints.models import Project, Product
+
+
+PAGES = [
+    '/',
+    '/projects',
+    '/products',
+    '/profile',
+    '/dashboard',
+    Product,
+    Project,
+]
+
+@api.route('/random_page')
+def get_random_page():
+    """Return a random URL on the frontend."""
+    random_page = random.choice(PAGES)
+    if isinstance(random_page, str):
+        return jsonify(url=random_page)
+
+    random_object = random_page.query.order_by(db.func.random()).limit(1).one_or_none()
```

Comment:
[This Stack Overflow question](https://stackoverflow.com/questions/5297396/quick-random-row-selection-in-postgres/32023533) and the first 2 answers suggest much more efficient alternatives

Reply:
Yeah, I've looked at them. They are significantly more cumbersome to implement, and this thing happens so rarely that it just really doesn't matter